presto-docs/src/main/sphinx/admin/properties.rst

```diff
@@ -843,6 +843,18 @@ Web UI Properties
 
 The following properties can be used to configure the :doc:`./web-interface`.
 
+``web-ui.authentication.type``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+    * **Type:** ``string``
+    * **Allowed values:** ``form``, ``fixed``, ``certificate``, ``kerberos``,
+      ``jwt``
+    * **Default value:** ``form``
+
+    The authentication mechanism to allow user access to the Web UI. Find more
+    informtion in the :ref:`Web UI authenication section
```

Comment:
Typos "informtion" and "authenication". These typos are duplicated below.

Shorten this to
```
See :ref:`Web UI Authentication <web-ui-authentication>`.

Reply:
Done in both places.